pkgs/build-support/setup-hooks/separate-debug-info.sh

```diff
@@ -1,7 +1,7 @@
 export NIX_SET_BUILD_ID=1
 export NIX_LDFLAGS+=" --compress-debug-sections=zlib"
 export NIX_CFLAGS_COMPILE+=" -ggdb -Wa,--compress-debug-sections"
-export NIX_RUSTFLAGS+=" -g"
+export NIX_RUSTFLAGS+=" -g -C strip=none"
```

Comment:
Should we add a comment why we disabled this?

Reply:
Isn't it pretty obvious that if you strip there won't be debug info?

Reply:
oh, I thought we where in the rust shell files. Yeah, probably.